cypress/integration/marc-authority/marc-authority-delete.spec.js

```diff
@@ -0,0 +1,75 @@
+import getRandomPostfix from '../../support/utils/stringTools';
+import { getLongDelay } from '../../support/utils/cypressTools';
+import TestTypes from '../../support/dictionary/testTypes';
+import Features from '../../support/dictionary/features';
+import DevTeams from '../../support/dictionary/devTeams';
+import Permissions from '../../support/dictionary/permissions';
+import TopMenu from '../../support/fragments/topMenu';
+import DataImport from '../../support/fragments/data_import/dataImport';
+import MarcAuthority from '../../support/fragments/marcAuthority/marcAuthority';
+import JobProfiles from '../../support/fragments/data_import/job_profiles/jobProfiles';
+import MarcAuthoritiesSearch from '../../support/fragments/marcAuthority/marcAuthoritiesSearch';
+import MarcAuthorities from '../../support/fragments/marcAuthority/marcAuthorities';
+import Users from '../../support/fragments/users/users';
+import MarcAuthoritiesDelete from '../../support/fragments/marcAuthority/marcAuthoritiesDelete';
+
+describe('MARC Authority management', () => {
+  const testData = {
+    userProperties: { name:'testname' },
+    uniqueFileName: `autotestFile.${getRandomPostfix()}.mrc`
+  };
+  const importFile = (profileName) => {
+    cy.visit(TopMenu.dataImportPath);
+
+    DataImport.uploadFile(MarcAuthority.defaultAuthority.name, testData.uniqueFileName);
+    JobProfiles.waitLoadingList();
+    JobProfiles.select(profileName);
+    JobProfiles.runImportFile(testData.uniqueFileName);
+    JobProfiles.openFileRecords(testData.uniqueFileName);
+    DataImport.getLinkToAuthority(MarcAuthority.defaultAuthority.headingReference).then(link => {
+      cy.intercept({
+        method: 'GET',
+        url: `/metadata-provider/jobLogEntries/${link.split('/').at(-2)}/records/${link.split('/').at(-1)}`,
+      }).as('getRecord');
+      cy.visit(link);
+      cy.wait('@getRecord', getLongDelay()).then(response => {
+        testData.internalAuthorityId = response.response.body.relatedAuthorityInfo.idList[0];
+
+        cy.visit(TopMenu.marcAuthorities);
+        MarcAuthoritiesSearch.searchBy('Uniform title', MarcAuthority.defaultAuthority.headingReference);
+        MarcAuthorities.select(testData.internalAuthorityId);
+        MarcAuthority.waitLoading();
+      });
+    });
+  };
+
+  beforeEach(() => {
+    cy.createTempUser([
+      Permissions.settingsDataImportEnabled.gui,
+      Permissions.moduleDataImportEnabled.gui,
+      Permissions.uiMarcAuthoritiesAuthorityRecordView.gui,
+      Permissions.uiMarcAuthoritiesAuthorityRecordEdit.gui,
+      Permissions.uiMarcAuthoritiesAuthorityRecordDelete.gui
+    ]).then(createdUserProperties => {
+      testData.userProperties.id = createdUserProperties.userId;
+      testData.userProperties.firstName = createdUserProperties.firstName;
+      testData.userProperties.name = createdUserProperties.username;
```

Comment:
u can write 1 row testData.userProperties = createdUserProperties; instead of 3 rows)

Reply:
done